### ISSUE TITLE
var scoping

### ISSUE THREAD

**Issue by mhenke** [mhenke/CFML-in-100-minutes#40]
Bit concerned that the CFC examples don't var the variable - that's kinda important to get into people's minds, especially if they're used to other languages...

**Comment by misterdai** [comment]
Agreed, should be important to teach proper scoping within CFC's.  Don't forget about standalone functions as well.

**Comment by mhenke** [comment]
A new section could be added or an existing section / example var scoping could be added
